pom.xml

```diff
@@ -61,6 +61,20 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
does that make sense to generate OSGi manifests for _all_ the modules in this project ?
for everything that's only gonna be deployed in the platform (and not be consumed by other projects) I doubt it's useful to generate those headers